src/Management/test/Endpoint.Test/ManagementWebHostBuilderExtensionsTest.cs

```diff
@@ -206,7 +194,7 @@ public async Task AddHeapDumpActuator_IWebHostBuilder_IStartupFilterFires()
     {
         if (Platform.IsWindows)
```

Comment:
Does this need to still be checking for `Platform.IsWindows`? I think at a minimum it works on Linux too

Reply:
I don't know, just left it as-is. Can you open an issue to investigate, if needed?

Reply:
opened #1361